lib/bitbucket_rest_api/repos.rb

```diff
@@ -10,7 +10,8 @@ class Repos < API
                  :Keys        => 'keys',
                  :Services    => 'services',
                  :Following   => 'following',
-                 :Sources     => 'sources'
+                 :Sources     => 'sources',
+                 :Forks       => 'forks'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.